common/scripted_effects/wc_artifacts_generic_scripted_effects.txt

```diff
@@ -750,56 +880,102 @@ game_start_spawn_artifacts_random_effect = { # Slots -> Types
-				3 = { add_artifact = spear_3 }
-				4 = { add_artifact = spear_2 }
-				5 = { add_artifact = spear_1 }
+				3 = { # Melee Weapons
+					trigger = { can_get_melee_weapon_at_gamestart_distribution_trigger = yes }
+					random_list = {
+						# Two-handed Swords
+						1 = { add_artifact = sword_twohanded_5 }
+						2 = { add_artifact = sword_twohanded_4 }
+						3 = { add_artifact = sword_twohanded_3 }
+						4 = { add_artifact = sword_twohanded_2 }
+						5 = { add_artifact = sword_twohanded_1 }
+						# One-handed Swords
+						1 = { add_artifact = sword_onehanded_5 }
+						2 = { add_artifact = sword_onehanded_4 }
+						3 = { add_artifact = sword_onehanded_3 }
+						4 = { add_artifact = sword_onehanded_2 }
+						5 = { add_artifact = sword_onehanded_1 }
+						# Two-handed Axes
+						1 = { add_artifact = axe_twohanded_5 }
+						2 = { add_artifact = axe_twohanded_4 }
+						3 = { add_artifact = axe_twohanded_3 }
+						4 = { add_artifact = axe_twohanded_2 }
+						5 = { add_artifact = axe_twohanded_1 }
+						# One-handed Axes
+						1 = { add_artifact = axe_onehanded_5 }
+						2 = { add_artifact = axe_onehanded_4 }
+						3 = { add_artifact = axe_onehanded_3 }
+						4 = { add_artifact = axe_onehanded_2 }
+						5 = { add_artifact = axe_onehanded_1 }
+						# Two-handed Maces
+						1 = { add_artifact = mace_twohanded_5 }
+						2 = { add_artifact = mace_twohanded_4 }
+						3 = { add_artifact = mace_twohanded_3 }
+						4 = { add_artifact = mace_twohanded_2 }
+						5 = { add_artifact = mace_twohanded_1 }
+						# One-handed Maces
+						1 = { add_artifact = mace_onehanded_5 }
+						2 = { add_artifact = mace_onehanded_4 }
+						3 = { add_artifact = mace_onehanded_3 }
+						4 = { add_artifact = mace_onehanded_2 }
+						5 = { add_artifact = mace_onehanded_1 }
+						# Halberds
+						1 = { add_artifact = halberd_5 }
+						2 = { add_artifact = halberd_4 }
+						3 = { add_artifact = halberd_3 }
+						4 = { add_artifact = halberd_2 }
+						5 = { add_artifact = halberd_1 }
+						# Spears
+						1 = { add_artifact = spear_5 }
+						2 = { add_artifact = spear_4 }
+						3 = { add_artifact = spear_3 }
+						4 = { add_artifact = spear_2 }
+						5 = { add_artifact = spear_1 }
+					}
+				}
+				1 = { # Ranged Weapons
+					trigger = { can_get_ranged_weapon_at_gamestart_distribution_trigger = yes }
+					random_list = {
+						1 = { # Bows
+							random_list = {
+								1 = { add_artifact = bow_5 }
+								2 = { add_artifact = bow_4 }
+								3 = { add_artifact = bow_3 }
+								4 = { add_artifact = bow_2 }
+								5 = { add_artifact = bow_1 }
+							}
+						}
+						1 = { # Crossbows
+							trigger = { crossbow_artifact_active_trigger = yes }
+
+							random_list = {
+								1 = { add_artifact = crossbow_5 }
+								2 = { add_artifact = crossbow_4 }
+								3 = { add_artifact = crossbow_3 }
+								4 = { add_artifact = crossbow_2 }
+								5 = { add_artifact = crossbow_1 }
+							}
+						}
+						1 = { # Rifles
```

Comment:
Just an idea. Shouldn't crossbows and rifles be more common than bows in the societies with high techologies?

Reply:
Done